components/brave_vpn/resources/panel/assets/svg-icons/waves-light.svg

```diff
@@ -0,0 +1,124 @@
+<svg width="314" height="317" viewBox="0 0 314 317" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
did you run this through svgomg.com? Seems to have a def referred by ID which may cause issues when inlined. Sometimes unavoidable, so just want to make sure.

Reply:
I didn't. I should though.